setupcfg2nix/cli.py

```diff
@@ -1,6 +1,56 @@
-from __future__ import print_function
 from setuptools.config import read_configuration
 import argparse
+import unittest.mock
+import errno
+import os
+import os.path
+import tempfile
+import setuptools
+import sys
+from distutils.errors import DistutilsFileError
+
+requires_sets = [ "install_requires", "setup_requires", "tests_require" ]
+
+def make_declarative(package_dir, out_file):
+    cur_dir = os.getcwd()
+    cur_path = sys.path
```

Comment:
Perhaps change this to `cur_path = sys.path[:]` and remove `sys.path = sys.path[:]` below.